src/azul/service/service_config/request_config.json

```diff
@@ -83,27 +87,30 @@
       "project.project_core.project_title": "project_title"
     },
     "contents.specimens": {
-      "donor_organism.sex": "biological_sex",
-      "donor_organism.biomaterial_core.biomaterial_id": "biomaterial_id",
       "specimen_from_organism.provenance.document_id": "document_id",
       "specimen_from_organism.diseases": "disease",
-      "specimen_from_organism.biomaterial_core.biomaterial_id": "donor_biomaterial_id",
-      "donor_organism.provenance.document_id": "donor_document_id",
-      "donor_organism.genus_species": "genus_species",
       "specimen_from_organism.organ": "organ",
       "specimen_from_organism.organ_part": "organ_part",
-      "donor_organism.organism_age": "organism_age",
-      "donor_organism.organism_age_unit": "organism_age_unit",
       "specimen_from_organism.preservation_storage.preservation_method": "preservation_method"
+    },
+    "contents.donors": {
+      "donor_organism.sex": "biological_sex",
+      "donor_organism.biomaterial_core.biomaterial_id": "biomaterial_id",
+      "donor_organism.provenance.document_id": "document_id",
+      "donor_organism.genus_species": "genus_species",
+      "donor_organism.organism_age": "organism_age",
+      "donor_organism.organism_age_unit": "organism_age_unit"
     }
```

Comment:
In the same way you added `content.donor`, please also include the `contents.organoid` and `contents.cell_line` fields here. Also include everything from `contents.sample` that you know the path of. For the inner entities already mentioned here, check that you list all their properties. The objective is for the manifest to include as many fields as possible.

Reply:
This is for `"manifest":`. The `"bdbag":` section is ok as is. I'm modifiying it in another PR and there is no need to introduce more potential for merge conflicts.